### ISSUE TITLE
Python binding is installed into a wrong path

### ISSUE THREAD

**Issue by yurivict** [jturney/ambit#54]
It is installed into ```lib/ambit/__init__.py``` when it should be installed into a sitelibdir. For example, ```/usr/local/lib/python3.8/site-packages```

**Comment by loriab** [comment]
Right, use the cmake variable pymod_install_libdir to provide the pythonx.x/site-packages spacer to imitate python install practices.
